client/my-sites/theme/main.jsx

```diff
@@ -240,6 +244,11 @@ const ThemeSheet = React.createClass( {
 		return <ThemeDownloadCard theme={ this.props.id } href={ this.props.download } />;
 	},
 
+	renderRelatedThemes() {
+
+		return <ThemeRelatedThemesCard currentTheme={ this.props.name } />;
```

Comment:
Unnecessary space here. If you get set up with [eslint](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#eslint), extra whitespace and unused imports will show up as warnings.